bank/Accounts/views.py

```diff
@@ -0,0 +1,44 @@
+from .models import Account
+from Accounts.serializer import AccountsSerializer, UserSerializer
+from rest_framework import generics
+from django.contrib.auth.models import User
+from rest_framework import permissions
+
+"""
+list all accounts or creates a new account
+"""
+
+
+class AccountsList(generics.ListCreateAPIView):
+    queryset = Account.objects.all()
+    serializer_class = AccountsSerializer
+
+
+""" 
```

Comment:
Docstrings should not be placed this way.  Please see: https://www.python.org/dev/peps/pep-0257/